### PR TITLE
Ewlj 137 poll results reveal

### PR DESCRIPTION
**Pull request by aCyborg**
## Ticket(s)

**Github Issue**

N/A

**Jira Ticket**

- [EWLj-137: Poll Results Reveal](https://issues.ama-assn.org/browse/EWLJ-137)

## Description

Creates poll functionality that reveals the results of the polls.

## To Test

- [ ] Spin up Patternlab
- [ ] review the poll organism
- [ ] review poll structured organism
- [ ] review their markup
- [ ] test the reveal/hide functionality.

## Visual Regressions
N/A

## Relevant Screenshots/GIFs
N/A

## Remaining Tasks
N/A

## Additional Notes
N/A

### CODE REVIEW

**Comment by aCyborg** [comment]
This is blocked by PR #17

**Review by gleroux02 — APPROVED** [review]
👍 once the other PR is merged this can be merged too.

**Pull request merged**
